src/main/webapp/about.jsp

```diff
@@ -58,11 +58,17 @@
       </ul>
 
       <p>
-        This is your code now. Get familiar with it and get comfortable
-        working with your team to plan and make changes. Start by updating the
-        homepage and this about page to tell your users more about your team.
-        This page should also be used to describe the features and improvements
-        you've added.
+        This chat web app is a product of the joint effort of 4 college 
```

Comment:
Fix the inconsistent whitespace here.

Reply:
Cool. I fixed the inconsistent white space.